examples/data_sharing.tex

```diff
@@ -0,0 +1,210 @@
+\documentclass[a4paper]{article}
+\usepackage{data}
```

Comment:
What package is this? I can't find it on CNET, unless I'm being an idiot (very possible).

Reply:
I made a new .sty file, because the existing ones already had ideas about sections and how to use them.

For the GDPR document, I redefined sections, subsections, and so on to look like normal text, so they could use and reference proper numbering automatically.

As mentioned elsewhere, perhaps a unified style file would be better?

We should probably get lawyer-input from @passamo9 about a specific number of styles for legal documents.  I don't know if it's good to have just one versatile style, or many bespoke style files.

Reply:
Please add that style file to your PR. We're using them to address those points, discussed in #30. Once this gets merged, the idea is to work through everything and shift it into its own `.cls` or `.sty`.

Reply:
Sorry - major malfunction on my end. The .sty file is in now.